email_app/app/utils/app_utils.js

```diff
@@ -65,15 +64,37 @@ export const deserialiseArray = (str) => {
   return Uint8Array.from(arrItems);
 }
 
-export const genKeyPair = () => {
```

Comment:
All these functions now should be moved to `safenet_comm.js` since they are part of the communication with safenet.